test/e2e/namespace_test.go

```diff
@@ -33,7 +33,14 @@ var _ = Describe("Podman namespaces", func() {
 	})
 
 	It("podman namespace test", func() {
-		SkipIfRemote("FIXME This should work on Remote")
+		if IsRemote() {
+			// --namespace flag not supported in podman remote
+			podman1 := podmanTest.Podman([]string{"--namespace", "test1", "run", "-d", ALPINE, "echo", "hello"})
+			podman1.WaitWithDefaultTimeout()
+			Expect(podman1.ExitCode()).To(Equal(125))
+			Expect(podman1.ErrorToString()).To(ContainSubstring("unknown flag: --namespace"))
+			return
+		}
```

Comment:
Silly question, but why not move the `if IsRemote()` down a couple of lines, and avoid duplicating the command & wait?
```go
    podman1.WaitWithDefaultTimeout()
    if IsRemote() {
        Expect(125) // and message
    } else {
        Expect(0)  // and message
    }
```

Reply:
Fixed